documentation/modules/cruise-control/proc-deploying-cruise-control.adoc

```diff
@@ -0,0 +1,91 @@
+// This module is included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-deploying-cruise-control-{context}']
+= Deploying Cruise Control
+
+Deploy Cruise Control through the `Kafka` resource in the `cruiseControl` properties.
```

Comment:
This is the wrong way round. We deploy CC throught the `cruiseControl` property of the `Kafka` resource?

Reply:
Also is "property" the correct term? Is it not a configuration?

Reply:
I think "property" is the correct term, it's not a k8s specific term but one we seem to use in our docs to refer to the fields/sections of a resource, I am happy to use "field" too. It is a configuration but I am worried it might get confused with the `cruiseControl.config` section specifically